llmstudio/engine/providers/azure.py

```diff
@@ -174,7 +175,9 @@ async def handle_tool_response(
 
         function_call_buffer = ""
         saving = False
+        normal_call_chunks = []
         for chunk in response:
+            chunk
```

Comment:
is this working? this line is not doing anything and this is not an inline condition.